battleship-server/src/main/java/com/epam/solid/nie/server/ShipSocketServer.java

```diff
@@ -35,29 +36,34 @@ public void initialize() throws IOException {
         second.register(serverSocket);
         players.add(second);
 
-        second.inform("Game has started. 1");
-        first.inform("Game has started. 2");
+        logger.info("Game has started. 1");
+        logger.info("Game has started. 2");
 
         String firstShips = first.provideShips();
-        System.out.println("First's ships:" + firstShips);
+        if (logger.isLoggable(Level.INFO)){
+            logger.info("First's ships:" + firstShips);
+        }
 
         String secondShips = second.provideShips();
-        System.out.println("Second's ships:" + secondShips);
+        if (logger.isLoggable(Level.INFO)){
+            logger.info("Second's ships:" + secondShips);
+        }
```

Comment:
And same question here. I'd provide ships for both players, then made one logger call, logging both fleets at once.